app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,8 @@
     <activity android:name=".testing.ContinuePlayingFragmentTestActivity" />
     <activity android:name=".home.HomeActivity" />
     <activity android:name=".player.audio.testing.AudioFragmentTestActivity" />
+    <activity android:name=".profile.AdminAuthActivity" />
+    <activity android:name=".profile.AddProfileActivity" />
```

Comment:
Nit: Try to keep activities in alphabetical order.

Reply:
Updated.